### ISSUE TITLE
Add category landing pages to improve SEO and communicate how WordPress is used.

### ISSUE THREAD

**Issue by StevenDufresne** [WordPress/wporg-showcase-2022#182]
I think categories provide the right context for shining the spotlight on how industry leaders are using WordPress.  I also think categories should be  the key navigational driver for the Showcase site and it should start on the homepage. I've scoped this ticket specific to category landing pages because we'll be able to measure it's effectiveness before making changes to the homepage.

**Acceptance Criteria**
- Categories should have landing page
- The landing page should have indexable content that relates to specific use cases so they are found when people search.
  -  “website status site examples”
  -  "website news examples"
- The landing pages should contain X showcase sites and link to the full archive

## Current Categories
We have an option to include multiple use cases in each category or add new category. I think we should keep with the current categories and add multiple use cases to each.

**Here are the current categories:**
- Business
- Creative
- Featured
- General
- Publication
- Store

Although we'll brainstorm together, here's a quick dump of what I'm thinking.

####  Business
**Use Cases**
- Startups
- Event logistics business
- etc...

####  General
**Use Cases**
- Status Pages
- Marketing Campaigns.
- ...etc

**Comment by ryelle** [comment]
We already do have category archive pages, ex https://wordpress.org/showcase-v2/category/business/ — those are the full archive for the category, and would stay the same?

But I think you're suggesting new pages (as in the CPT page) per category with some actual content? including a small grid of sites, like the homepage has.

(I hope that's what you mean at least, because that would be much easier than conditionally changing the archive templates, and could be rolled out pretty much without dev help)

**Comment by StevenDufresne** [comment]
> But I think you're suggesting new pages (as in the CPT page) per category with some actual content? including a small grid of sites, like the homepage has.

Yes. We wouldn't need to change the current archives. Good point.
